operator/scripts/download-controller-gen.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+CONTROLLER_GEN_TMP_DIR=$(mktemp -d)
+cd $CONTROLLER_GEN_TMP_DIR
+
+go mod init tmp
+echo "Downloading controller gen"
+go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1
```

Comment:
I think a `go install` would work better here. in 1.16, an `install` won't modify the project's `go.mod` or `go.sum` file, which is nice. what do you think?

Reply:
Yeah it'd be better, I'll change it